.gitignore

```diff
@@ -1,3 +1,5 @@
+\#*#
+.#*
```

Comment:
Should it be:
```suggestion
\#*\#
.\#*
```

Reply:
Actually not, because `#` is only a comment character at the beginning. Here is a quote from the manual (from section PATTERN FORMAT).

> A line starting with # serves as a comment. Put a backslash ("\") in front of the first hash for patterns that begin with a hash.

In particular, it does not say anything about backslashes at other locations than the first.

Reply:
Thank you for pointing to the manual. It seems that number of places on internet copied patterns from a single source with unnecessary slash.